src/mix/api/channels-types.ts

```diff
@@ -21,4 +23,53 @@ export type ChannelsRenameBodyParams = {
   displayName: string;
 }
 
+/** @hidden */
+export type ChannelsCreatePathParams = {
+  /** ID of the relevant project. */
+  projectId: string;
+}
+
+/** @hidden */
+export type ChannelsConfigPathParams = ChannelsCreatePathParams;
+
+// REVIEW: this should almost certainly be put in its own file
+export const ChannelModalities = [
```

Comment:
If you look at how I ended up doing DataTypes in entities-types, the keys are what the UI expects and the values are what the endpoint expects. Something like that may help here.